lib/msal-browser/src/config/Configuration.ts

```diff
@@ -51,13 +51,15 @@ export type CacheOptions = {
  * - windowHashTimeout            - sets the timeout for waiting for a response hash in a popup
  * - iframeHashTimeout            - sets the timeout for waiting for a response hash in an iframe
  * - loadFrameTimeout             - maximum time the library should wait for a frame to load
+ * - delayOpenPopup               - Sets whether popup is open as soon as loginPopup is called. By default, this flag is set to false. Set to true for native apps and PWA.
  */
 export type BrowserSystemOptions = SystemOptions & {
     loggerOptions?: LoggerOptions;
     networkClient?: INetworkModule;
     windowHashTimeout?: number;
     iframeHashTimeout?: number;
     loadFrameTimeout?: number;
+    delayOpenPopup?: boolean;
```

Comment:
Note: Open to suggestions for a better name for this?

Reply:
`asyncPopups` is the only goodish name I can think of at the moment.